dashboard/src/api/hooks/useDeleteVersion.ts

```diff
@@ -0,0 +1,29 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { GetVersionsResponse } from '../../interfaces/versions';
+import client from '../client';
+import '../mocks/useDeleteVersion.mock';
+
+const deleteVersion = async (projectId?: number, versionId?: number) => {
+  const response = await client.delete<GetVersionsResponse>(
+    `/api/v1/projects/${projectId}/versions/${versionId}`,
```

Comment:
there are a lot of places where we call /api/v1/projects -> create this as a constant instead.

Reply:
you mean something like const projects = '/api/v1/projects'?